package.json

```diff
@@ -17,19 +17,20 @@
     "zod": "^4.1.4"
   },
   "scripts": {
-    "compose": "node ./compose.js",
-    "locales:prepare": "node scripts/translations/prepare.js",
-    "locales:download": "node scripts/translations/download-locales.mjs",
-    "locales:upload": "node scripts/translations/upload-base-locale.mjs",
-    "compress": "node scripts/repository/compress.js",
+    "compose": "node scripts/check/check-node-version.js && node ./compose.js",
```

Comment:
can't we import this check inside of the scripts?
thus commands here would look better